bundles/org.openhab.binding.telegram/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This looks like a combination of spaces and tabs

Reply:
Tabs are the preferered option here

Reply:
fixed